charts/cactus-backend/values.yaml

```
@@ -155,14 +179,128 @@ persistentVolume:
   ##
   # volumeName: ""
 
-emptyDir:
-  ## emptyDir volume size limit
-  ##
-  sizeLimit: ""
-
+# https://github.com/bitnami/charts/tree/postgresql/16.7.27/bitnami/postgresql#parameters
+postgresql:
+  # If true, the PostgreSQL dependency is enabled
+  enabled: true
+  image:
+    repository: bitnamilegacy/postgresql
+  auth:
+    username: cactus-backend
+    password: cactus-backend
+    database: cactus-backend
+  primary:
+    networkPolicy:
+      enabled: false
+    resources:
+      limits:
+        cpu: 1
+        memory: 256Mi
+        ephemeral-storage: 2Gi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+        ephemeral-storage: 50Mi
+  fullnameOverride: cactus-backend-postgresql
+
+# https://github.com/bitnami/charts/tree/redis/22.0.7/bitnami/redis#parameters
+redis:
+  # If true, the Redis dependency is enabled
+  enabled: true
+  image:
+    repository: bitnamilegacy/redis
+  metrics:
+    enabled: true
+    image:
+      repository: bitnamilegacy/redis-exporter
+    resources:
+      limits:
+        cpu: 200m
+        memory: 32Mi
+      requests:
+        cpu: 100m
+        memory: 32Mi
+    serviceMonitor:
+      enabled: false
+      namespace: monitoring
+  master:
+    resources:
+      limits:
+        cpu: 1
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+    disableCommands:
+      - FLUSHALL
+      - FLUSHDB
+  replica:
+    replicaCount: 1
+    resources:
+      limits:
+        cpu: 1
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+    disableCommands:
+      - FLUSHALL
+      - FLUSHDB
+  auth:
+    enabled: true
+    password: cactus-backend
+  fullnameOverride: redis
+
+postgresqlha:
+  enabled: false
 
-# These values determine which keys inside the Kubernetes secret that contain the contents
-# of the configs/config.json and parsers/config.json files that are mounted by the chart:
+# See https://github.com/juicedata/charts/blob/helm-chart-juicefs-s3-gateway-0.11.3/charts/juicefs-s3-gateway/values.yaml
+juicefs-s3-gateway:
+  enabled: false
 
-# generalConfigKey: cactus-backend-general-config
-# clientParserConfigKey: cactus-backend-client-parser-config
+emptyDir:
+  sizeLimit: 8Gi
+
+envVars:
+  GATEWAY_PORT: 8080
+  GATEWAY_ADMIN_PORT: 18190
+  ENV: prod # Anything than `test`
```

```suggestion
  ENV: prod # Anything other than 'test'
```
Do we default to `test`? Otherwise we can remove it maybe.